pudb/var_view.py

```diff
@@ -285,15 +287,15 @@ def type_stringifier(value):
         try:
             return text_type(value)
         except Exception:
-            pass
+            ui_log.exception('string safe type stringifier failed')
```

Comment:
Perhaps update `result` to reflect a brief version of the error? (And point to the internal shell for traceback?)